terraform/helm/genesis/files/genesis.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#
+# Runs an automated genesis ceremony for validators spun up by the aptos-node helm chart
+#
+# Expect the following environment variables to be set before execution:
+# NUM_VALIDATORS
+# ERA
+# WORKSPACE: default /tmp
+# USERNAME_PREFIX: default aptos-node
+# VALIDATOR_HOST_SUFFIX: default validator-lb
+# FULLNODE_HOST_SUFFIX: default fullnode-lb
+# 
+
+WORKSPACE=${WORKSPACE:-/tmp}
+USERNAME_PREFIX=${USERNAME_PREFIX:-aptos-node}
+VALIDATOR_HOST_SUFFIX=${VALIDATOR_HOST_SUFFIX:-validator-lb}
+FULLNODE_HOST_SUFFIX=${FULLNODE_HOST_SUFFIX:-fullnode-lb}
+
+if [[ -z ${ERA} ]] || [[ -z ${NUM_VALIDATORS} ]]; then
+    echo "ERA (${ERA:-null}) and NUM_VALIDATORS (${NUM_VALIDATORS:-null}) must be set"
+    exit 1
+fi
+
+echo "NUM_VALIDATORS=${NUM_VALIDATORS}"
+echo "ERA=${ERA}"
+echo "WORKSPACE=${WORKSPACE}"
+echo "USERNAME_PREFIX=${USERNAME_PREFIX}"
+echo "VALIDATOR_HOST_SUFFIX=${VALIDATOR_HOST_SUFFIX}"
+echo "FULLNODE_HOST_SUFFIX=${FULLNODE_HOST_SUFFIX}"
+
+# generate all validator configurations
+for i in $(seq 0 $(($NUM_VALIDATORS-1))); do
+username="${USERNAME_PREFIX}-${i}"
+user_dir="${WORKSPACE}/${username}"
+mkdir $user_dir
+aptos genesis generate-keys --output-dir $user_dir
+aptos genesis set-validator-configuration --keys-dir $user_dir --local-repository-dir $WORKSPACE \
+    --username $username \
+    --validator-host "${username}-${VALIDATOR_HOST_SUFFIX}:6180" \
+    --full-node-host "${username}-${FULLNODE_HOST_SUFFIX}:6182"
+done
+
+# get the framework
+# this is the directory the aptos-framework is located in the aptoslabs/init docker image
```

Comment:
maybe simplify this to "Copy Aptos Framework from the aptoslabs/init docker image"